src/pull_request.ts

```diff
@@ -278,3 +282,25 @@ async function submitReview(
     );
   }
 }
+
+function shouldIgnorePullRequest(pull_request: { body?: string }) {
+  const ignorePhrases = [
+    "@presubmit ignore",
+    "@presubmit: ignore",
+    "@presubmit skip",
+    "@presubmit: skip",
+    "@presubmitai ignore",
+    "@presubmitai: ignore",
+    "@presubmitai skip",
+    "@presubmitai: skip",
+  ];
+  const bodyLower = (pull_request.body ?? "").toLowerCase();
+
+  for (const phrase of ignorePhrases) {
+    if (bodyLower.includes(phrase.toLowerCase())) {
+      info(`ignoring pull request because of '${phrase}' in description`);
+      return true;
+    }
+  }
```

Comment:
The code could be simplified and made more efficient using the `some()` array method. Additionally, `toLowerCase()` is being called unnecessarily on each iteration for constant strings. Consider:
```typescript
const ignorePhrases = [
  '@presubmit ignore',
  // ... other phrases
].map(phrase => phrase.toLowerCase());

return ignorePhrases.some(phrase => {
  const matches = bodyLower.includes(phrase);
  if (matches) {
    info(`ignoring pull request because of '${phrase}' in description`);
  }
  return matches;
});
```